metadata/gc_test.go

```diff
@@ -402,22 +402,6 @@ func addLeaseContent(ns, lid string, dgst digest.Digest) alterFunc {
 	}
 }
 
-func addContainer(ns, name, snapshotter, snapshot string, labels map[string]string) alterFunc {
```

Comment:
@dmcgowan Possible missing test case here? Do containers not pin resources?

Reply:
They do, this should be used to add to test case. Will make a PR for that